### PR TITLE
Poc/naive bayes classifier [Supervised Learning]

### PR DESCRIPTION
**Pull request by simonprovost**
Naive Bayes Classifier on Awake data. How to classify a raw data _t_ as awake or asleep via probabilistic supervised learning.

### CODE REVIEW

File: poc/NaiveBayesClassifier/readme.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+setuptools      51.1.1 // Do not sure it's relevant for this POC but it was in my list.
+six             1.15.0 // Do not sure it's relevant for this POC but it was in my list.
+threadpoolctl   2.1.0 // Do not sure it's relevant for this POC but it was in my list.
+wheel           0.36.2 // Do not sure it's relevant for this POC but it was in my list.
+
+## POC Description
+A Naive bayes classification algorithm implementation on Awake Data. Using the Gaussian Naive bayes (why? https://www.quora.com/What-is-the-difference-between-the-the-Gaussian-Bernoulli-Multinomial-and-the-regular-Naive-Bayes-algorithms).
+
+## POC Report (Training model Explanations, Results (score, confusion matrix))
+https://docs.google.com/document/d/1Ips63u-SlEB1M4ZSCyfKwf-1wIwfsGS8VTCKZ1upmmE/edit?usp=sharing
```
Comment (Leodau) [inline poc/NaiveBayesClassifier/readme.md:36]: This repo is public, you are sure you wanna give access to this document? :thinking:

File: poc/NaiveBayesClassifier/algorithm.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+##### Print model viability/score
+
+#print(model.score(X_test, y_test))
+#print(y_test[:10])
+#print(model.predict(X_test[:10]))
+
+##### Save the model as a pickle in a file
+#if not (sys.version_info.major == 3 and sys.version_info.minor >= 5):
+    #joblib.dump(model, 'modelSaved.pkl')
```
Comment (Leodau) [inline poc/NaiveBayesClassifier/algorithm.py:36]: Why so many commented forgotten lines?

**Review by Leodau — APPROVED** [review]

File: poc/NaiveBayesClassifier/readme.md (hunk 1 above)
Comment (simonprovost) [reply poc/NaiveBayesClassifier/readme.md:36]: Very good point @Leodau. I am on my way to give you a fixup on that. The code could be open to the world as it is not Awake Algorithm. However, the right point is that, every report available on our POCs should be protected by a password.

File: poc/NaiveBayesClassifier/algorithm.py (hunk 2 above)
Comment (simonprovost) [reply poc/NaiveBayesClassifier/algorithm.py:36]: As I have mentioned previously in the Readme.md file, there is an issue with saving your model as it asks your Python version to be downgraded (v:2.7). On the other hand, using confusion matrices in python 2.7 is not supported anymore. So, I had to make a choice for the following POC: Which was that, I am going to deliver the version with confusion matrix and Python 3+, where people with the wish to try the save/load feature will even be able to by removing comments, and obviously downgrading their python version (see Readme.MD). Would you like me, to edit/remove a bit for you?

@Leodau

File: poc/NaiveBayesClassifier/readme.md (hunk 1 above)
Comment (simonprovost) [reply poc/NaiveBayesClassifier/readme.md]: I've been taking care to remove the previous document and add (via a `fixup`commit see on the GitHub PR) a new one with protected password. As soon as you accept the new approach, keep me updated and I will autosquash the fixup (However, if you want to do it, feel free to proceed the autosquash).

**Pull request merged**